test/db/cmd/types

```diff
@@ -1260,7 +1260,7 @@ var char * var_28h @ rbp-0x28
 var char * var_20h @ rbp-0x20
 var int64_t var_18h @ rbp-0x18
 var int64_t var_8h @ rbp-0x8
-arg char ** arg2 @ rsi
+arg char * arg2 @ rsi
```

Comment:
is this change correct?

Reply:
It is a slightly worse result for how the code looks here but with respect to how the algorithm works it is correct, so yes it's a fix.